src/tables/sortable-table.js

```diff
@@ -47,39 +51,43 @@ const propTypes = {
   onChange: PropTypes.func,
   rowComponent: Types.component,
   headerComponent: Types.component,
-}
+  ...Caption.propTypes,
+  caption: PropTypes.string,
```

Comment:
Per [David's request](https://github.com/LaunchPadLab/lp-components/issues/463#issuecomment-756344487), we should make this a `node`